torch_frame/data/dataset.py

```diff
@@ -316,7 +336,15 @@ def materialize(self, device: Optional[torch.device] = None,
 
         # 1. Fill column statistics:
         for col, stype in self.col_to_stype.items():
-            self._col_stats[col] = compute_col_stats(self.df[col], stype)
+            if stype == torch_frame.multicategorical:
```

Comment:
multi_categorical

Reply:
I think it's called `multicategorical` inside Kumo. We should probably align those two.

Reply:
I see. I still think `multi_categorical` is better. It's not difficult to align internally.